libcontainer/system/proc.go

```diff
@@ -75,29 +78,42 @@ func parseStat(data string) (stat Stat_t, err error) {
 	// From proc(5), field 2 could contain space and is inside `(` and `)`.
 	// The following is an example:
 	// 89653 (gunicorn: maste) S 89630 89653 89653 0 -1 4194560 29689 28896 0 3 146 32 76 19 20 0 1 0 2971844 52965376 3920 18446744073709551615 1 1 0 0 0 0 0 16781312 137447943 0 0 0 17 1 0 0 0 0 0 0 0 0 0 0 0 0 0
-	i := strings.LastIndex(data, ")")
+	i := strings.LastIndexByte(data, ')')
```

Comment:
> wondering if passing data to parseStat() as []byte (not converting it to a string) could optimise even more

I was thinking about that, too, but decided to not go down that road (thought it will decrease readability and result in a few smaller allocations instead of a single bigger one, which is not necessarily makes things better).